tcf_website/static/course/javascript/course_data.js

```diff
@@ -45,6 +90,15 @@ const loadData = data => {
 };
 
 const createChart = gradesData => {
+    // 1. Justification for no-new: (Do not use 'new' for side effects)
+    // Without disabling the warning, eslint complains about using `new` to produce side-effects.
+    // (Which is how chart.js works. We can't change that.)
+    // You can silence it by assigning the expression to a variable. But then, eslint complains that we have an unused variable.
+    // We're not going to be able to avoid this, so I've disabled the error.
+    // 2. Justification for no-undef: ('Chart' is not defined)
+    // We could avoid this in the future by using WebPack or plain old ES6 modules.
+    // But right now, the chart.js source is referenced in the templates themselves through a CDN,
+    // so eslint will always complain. We'll just silence it.
```

Comment:
I could be wrong, but I think this comment section was explaining the use of eslint that was moved to line 259, so it might be best to move this section to right before that line